.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+### Vision
+
+What do we want out of this feature?
```

Comment:
For nomenclature, it might be better to say "story" rather than "feature". We can use this for new features as well as bugs 🐛